site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -369,3 +389,15 @@ const fillNameAndDisplayWithFilename = async (
     form.setFieldValue("display_name", capitalize(name)),
   ]);
 };
+
+const ProvisionerWarning: FC = () => {
+  return (
+    <Alert severity="warning" css={{ marginBottom: 16 }}>
+      This organization does not have any provisioners. Before you create a
+      template, you&apos;ll need to configure a provisioner.{" "}
+      <Link href={docs("/admin/provisioners#organization-scoped-provisioners")}>
+        See our documentation.
+      </Link>
+    </Alert>
+  );
+};
```

Comment:
Can you drop a comment that this banner will not go away if a provisioner becomes available? Just note their is no polling to update state, so do not change the form based on this state